### ISSUE TITLE
[BUG][  script:qb-lapraces] SCRIPT ERROR: @qb-lapraces/server/main.lua:238: attempt to index a nil value (field '?')

### ISSUE THREAD

**Issue by josephlees123** [qbcore-framework/qb-lapraces#39]
[  script:qb-lapraces] SCRIPT ERROR: @qb-lapraces/server/main.lua:238: attempt to index a nil value (field '?') can someone please drop a working qb-lapraces please only 1 script  i have been trying to get working for 3 days and can see the problem

**Comment by kewlkitty** [comment]
getting the same thing, has this been addressed? script:qb-lapraces] SCRIPT ERROR: @qb-lapraces/server/main.lua:238: attempt to index a nil value (field '?')

**Comment by Rhydium** [comment]
Same issue here, edited nothing at all and having this error

**Comment by Rhydium** [comment]
I was able to fix this by the fix that user RubbishPanda#6266 on Discord posted:

In **qb-phone/html/js/racing.js**, on line 135 you'll see the following code:
```
$.post('https://qb-phone/JoinRace', JSON.stringify({
    RaceData: {
        ...Data,
        RaceId
    }
}));
```

Replace this with:
```
$.post('https://qb-phone/JoinRace', JSON.stringify({
    RaceData: Data,
}));
```
